### PR TITLE
Bump Mattermost to 5.18

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
- Update Mattermost dependency to v5.18
- Update MM import path using `find . -name "*.go" -type f | xargs sed -i -e 's/github.com\/mattermost\/mattermost-server\//github.com\/mattermost\/mattermost-server\/v5\//g'`
- Run `go mod tidy`

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-21354

### CODE REVIEW

File: server/plugin.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (p *Plugin) OnActivate() error {
 
 // SendEphemeralPost sends a ephemeral message in a given channel to a given user
 func (p *Plugin) SendEphemeralPost(channelID, userID, message string) {
-	// This is mostly taken from https://github.com/mattermost/mattermost-server/blob/master/app/command.go#L304
+	// This is mostly taken from https://github.com/mattermost/mattermost-server/v5/blob/master/app/command.go#L304
```
Comment (agnivade) [inline server/plugin.go]: This is incorrect :P

**Review by agnivade — CHANGES_REQUESTED** [review]
Please also mention the command used to make these changes. You had a `sed` command right ?

And lastly, I assume you ran a "go mod tidy" on this ?

File: server/plugin.go (hunk 1 above)
Comment (hanzei) [reply server/plugin.go]: Good catch :+1:

**Comment by hanzei** [comment]
Good suggestions @agnivade :+1:

**Review by agnivade — APPROVED** [review]

**Review by cpoile — APPROVED** [review]

**Comment by cpoile** [comment]
@hanzei QA review, or no?

**Comment by hanzei** [comment]
Not worth for this plugin :wink:

**Pull request merged**